extensions/amp-video/1.0/test/validator-amp-video-cached-source.html

```diff
@@ -0,0 +1,3 @@
+/*# sourceURL=/css/ampshared.css*/</style><script async src="https://cdn.ampproject.org/v0.mjs" type="module" crossorigin="anonymous"></script><script async nomodule src="https://cdn.ampproject.org/v0.js" crossorigin="anonymous"></script><script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.mjs" type="module" crossorigin="anonymous"></script><script async nomodule src="https://cdn.ampproject.org/v0/amp-video-0.1.js" crossorigin="anonymous" custom-element="amp-video"></script><title>AMP #0</title><link rel="canonical" href="amps.html"><style amp-custom></style></head><body> <amp-video autoplay width="720" height="405" layout="responsive" class="i-amphtml-layout-responsive i-amphtml-layout-size-defined" i-amphtml-layout="responsive"><i-amphtml-sizer style="display:block;padding-top:56.25%"></i-amphtml-sizer> <source src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerJoyrides.mp4" type="video/mp4" i-amphtml-video-cached-source> </amp-video>
```

Comment:
Nit: I would add a html `<!-- comment -->` explaining that we expect this to fail.

Reply:
Done